src/app/app.module.ts

```diff
@@ -50,7 +50,7 @@ export function hljsLanguages(): { [name: string]: () => Promise<any> } {
   ],
   imports: [
     BrowserModule,
-    RouterModule.forRoot(routes),
+    RouterModule.forRoot(routes, { relativeLinkResolution: 'legacy' }),
```

Comment:
Have you tested disabling this strategy? I believe schematics have it like this for safety, but should not impact this project. Try deleting `{ relativeLinkResolution: 'legacy' }`